Simulate.Tests/Tests.cs

```diff
@@ -44,6 +44,21 @@ public async Task Should_Fail_Correctly_By_Default()
         Assert.IsTrue(results.Failures == 1);
     }
 
+    [TestMethod]
+    [Description("Ensures that simulations that throw fail correctly without options.")]
+    public async Task Uncaught_Exception_Should_Fail_Correctly_By_Default()
+    {
+
+        var results = await new Simulation("test", () =>
+        {
+            throw new Exception();
+        })
+        .Run();
+
+        Assert.IsTrue(results.Successes == 0);
```

Comment:
```suggestion
        Assert.AreEqual(0, results.Successes);
        Assert.AreEqual(1, results.Failures);
```

Dan krijg je een mooiere foutmelding omdat hij dan onderscheid kan maken tussen de expected en actual value, anders weet hij alleen of het true of false is.